docs/modules/installation.rst

```diff
@@ -0,0 +1,117 @@
+Installing Protons
+==================
+
+Follow these instructions to install ``protons``, and its dependencies.
+If you run into any issues, please `raise an issue on our Github page`_.
+
+Install using conda
+-------------------
+
+The recommended way to install ``protons`` will be through conda.
+You can obtain conda by installing the Anaconda_ python distribution.
+It will provide a wide suite of scientific python packages, including some of the dependencies in ``protons``.
+Additionally it provides the very powerful ``conda`` python environment and package manager.
+For instructions on how to use conda, please `see this page`_ to get you started.
+
+As part of the first release, we will start building conda packages for ``protons``.
+You will be able to install ``protons`` using the following conda command.
+
+.. code-block:: bash
+
+  conda install -c omnia protons
+
+This will install the latest version of ``protons`` from the Omnia_ channel.
+
+.. _Omnia: http://www.omnia.md/
+.. _Anaconda: https://www.continuum.io/why-anaconda
+.. _see this page: http://conda.pydata.org/docs/get-started.html
+.. Note::
+
+   Currently, no conda packages are being built.
+   You can install dependencies through conda, but the package itself needs to be installed using ``setup.py`` .
+   See below for instructions.
+
+Install using setup.py
+----------------------
+
+Currently, the only way to install protons is by using the ``setup.py`` script that is provided with the package.
+
+Download a copy of the repository, and cd into it.
+
+.. code-block:: bash
+
+    git clone https://github.com/choderalab/constph-openmm.git
+    cd constph-openmm
+
+Them, use the command
+
+.. code-block:: bash
+
+    python setup.py install
+
+to install the package. The ``setup.py`` installation does not automatically check for requirements.
+Make sure to install the dependencies separately using ``pip``, or ``conda``. See below for instructions.
+
+Requirements
+------------
+
+The ``protons`` package has the following requirements:
+
+    - python 2.7, 3.4 or 3.5
+    - openmm_ >=7.0.1
+    - numpy_ >=1.10
+    - scipy_ >=0.17.0
+    - openmmtools_ >=0.7.5
+    - pymbar_
+    - openmoltools_ >=0.7.0
+    - ambermini_
+    - joblib_
+    - lxml_
+
+These should be available from the `omnia` conda channel.
+The following example commands create a new conda environment called "protons", with all dependencies installed.
+
+.. code-block:: bash
+
+    # Point conda to the omnia channel as package source
+    conda config --add channels omnia
+    # Create a new environment called "protons", with all dependencies included
```

Comment:
I tried setting up a requirements file by exportign an environment, but I'm getting errors.

-bash-4.1$ conda env create -f protons.yml
Fetching package metadata .........
Solving package specifications: .
Error: Packages missing in current linux-64 channels:
  - omnia::ambermini 15.0.4 5
  - omnia::fftw3f 3.3.4 2
  - omnia::mdtraj 1.7.2 np110py35_0
  - omnia::openmm 7.0.1 py35_0
  - omnia::openmmtools 0.7.5 py35_1
  - omnia::openmoltools 0.7.1 py35_4
  - omnia::parmed 2.6.1 py35_0
  - omnia::pymbar 3.0.0.beta2 np110py35_0

Not sure why, because I just installed the exact environment from scratch and exported it. I'm going to leave this as is, since my current instructions work.